demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -240,6 +242,50 @@ def verify_readme_image_paths(self) -> bool:
             return False
         return True
 
+    def verify_readme_relative_urls(self) -> bool:
+        """ Validate readme (not pack readme) relative urls.
+
+                Returns:
+                    bool: True If there are no invalid relative urls.
+                """
+        # If there are errors in one of the following validations return False
+        error_list = self.check_readme_relative_url_paths()
+        if len(error_list):
+            return False
+        return True
+
+    @error_codes('RM112')
+    def check_readme_relative_url_paths(self, is_pack_readme: bool = False) -> list:
+        """ Validate readme url relative paths.
+            prints an error if relative paths in README are found since they are not supported.
+
+        Arguments:
+            is_pack_readme (bool) - True if the the README file is a pack README, default: False
+
+        Returns:
+            list: List of the errors found
+        """
+        error_list = []
+        # If error was found, print it only if its not a pack readme. For pack readme, the PackUniqueFilesValidator
+        # class handles the errors and printing.
+        should_print_error = not is_pack_readme
+        relative_urls = re.findall(RELATIVE_MARKDOWN_URL_REGEX, self.readme_content,
+                                   re.IGNORECASE | re.MULTILINE)
+        relative_urls += re.findall(RELATIVE_HREF_URL_REGEX, self.readme_content,
+                                    re.IGNORECASE | re.MULTILINE)
+        for url in relative_urls:
+            # skip empty links
+            if not url[1]:
```

Comment:
this means that we found an empty url right? Lets make that clear with a comment

Reply:
are you sure this shouldnt be [2] ?

Reply:
There is the following comment
`            # skip empty links`
Should it be changed? 
And yes the regex has only 2 results in it, to explain better-
the link - `[lala test]()`
is converted to the following -
<img width="423" alt="Screen Shot 2022-06-19 at 12 35 57" src="https://user-images.githubusercontent.com/86190583/174474884-5f61012e-f691-49d1-bd8e-7dcf8c03ab2c.png">

Reply:
got confused with re.finditer that returns the full match as 0 (might be worth using for replacement purpose instead of rebuilding the string